src/sentry/models/project.py

```diff
@@ -260,12 +260,6 @@ class flags(TypedClassBitField):
         # This project has sent feedbacks
         has_feedbacks: bool
 
-        # spike_protection_error_currently_active
-        spike_protection_error_currently_active: bool
-
-        spike_protection_transaction_currently_active: bool
-        spike_protection_attachment_currently_active: bool
-
```

Comment:
This won't be safe with these being in the middle of the bit field. Django will now map the other bits into the slots that these options were using and effectively scramble the rest of the flags :cry:

You could rename these bits to have `unused` prefixes to indicate that they are dead bits. :shrug:

Reply:
😭